src/Servers/Kestrel/Core/src/Internal/Http3/Http3Connection.cs

```diff
@@ -459,9 +459,9 @@ private async ValueTask ProcessOutboundControlStreamAsync(Http3ControlStream con
             features.Set<IStreamDirectionFeature>(new DefaultStreamDirectionFeature(canRead: false, canWrite: true));
             var streamContext = await _multiplexedContext.ConnectAsync(features);
             var httpConnectionContext = new Http3StreamContext(
-                connectionId: streamContext.ConnectionId,
+                _multiplexedContext.ConnectionId,
                 HttpProtocols.Http3,
-                connectionContext: null!, // TODO connection context is null here. Should we set it to anything?
+                streamContext,
```

Comment:
Isn't `streamContext` supposed to be the third-to-last argument and this supposed to be the MultiplexedConnectionContext?